tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,197 @@
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   1
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    2001
+    user clicks button   Update status
+
+    user waits until h2 is visible  Sign off
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${day} ${month_word} ${year}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+
+Go to Public release page and ensure release isn't visible
+    [Tags]  HappyPath
+    user goes to url  %{PUBLIC_URL}
+    user goes to url  ${PUBLIC_RELEASE_LINK}
+
+Go to Table Tool page
+    [Tags]  HappyPath
+    environment variable should be set  PUBLIC_URL
+    user goes to url  %{PUBLIC_URL}/data-tables
+    user waits until h1 is visible  Create your own tables online
+
+Check scheduled release isn't visible
+    [Tags]  HappyPath
+    environment variable should be set  TEST_THEME_NAME
+    user opens details dropdown    %{TEST_THEME_NAME}
+    user checks page does not contain  UI test topic %{RUN_IDENTIFIER}
+
+Go to release URL and check release isn't visible
+    user goes to url  ${PUBLIC_RELEASE_LINK}
+    [Tags]  HappyPath NotAgainstLocal
+    user waits for page to finish loading
+    user checks page contains  Page not found
+
+Return to admin dashboard
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+
+Go to release from admin dashboard
+    [Tags]  HappyPath
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown  ${DATE}  ${accordion}
+    ${details}=  user gets details content element  ${DATE}  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="Edit this release"]
+    user clicks link  Edit this release
+
+Approve release for immediate publication
+    [Tags]  HappyPath
+    user clicks link  Sign off
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+    user clicks radio  Ready for higher review
+    user clicks radio  Approved for publication
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by UI tests
+    user clears element text  id:releaseStatusForm-publishScheduled-day 
+    user clears element text  id:releaseStatusForm-publishScheduled-month 
+    user clears element text  id:releaseStatusForm-publishScheduled-year
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+
+Wait for release process status to be Complete
```

Comment:
I'd rename and change this test case so that it checks the release hasn't been published. So verify that the "There is a problem` text appears on the page after the `Update status` button has been clicked and maybe check that Approved doesn't appear on the page.